schematic/schemas/data_model_edges.py

```diff
@@ -0,0 +1,64 @@
+import networkx as nx
+
+from schematic.schemas.data_model_relationships import (
+    DataModelRelationships
+    )
+
+class DataModelEdges():
+    def __init__(self):
+        self.dmr = DataModelRelationships()
+        self.data_model_relationships = self.dmr.relationships_dictionary
+
+    def generate_edge(self, G: nx.MultiDiGraph, node: str, all_node_dict: dict, attr_rel_dict: dict, edge_relationships: dict) -> nx.MultiDiGraph:
+        """Generate an edge between a target node and relevant other nodes the data model. In short, does this current node belong to a recorded relationship in the attribute, relationshps dictionary. Go through each attribute and relationship to find where the node may be.
+        Args:
+            G, nx.MultiDiGraph: networkx graph representation of the data model, that is in the process of being fully built.
```

Comment:
I would add to the description to say where it is in the process of being built: ex "A graph representation of the data model that has all nodes added but none/some edges"